pset-iterators.js

```diff
@@ -184,6 +232,13 @@ console.log('Problem 7:')
 
 // Add your code below this line
 
+const shortTermPresidents = presidents.filter((objectPresident) => {
+  const lessThanFourYears = objectPresident.took_office.split(',')
+  return lessThanFourYears.length <= 4
```

Comment:
This logic seems to return every president, as .length returns the number of elements or characters; [here](https://github.com/jsd20200127/hw-03-iterators-solution/blob/c31d6d65d8f18a945cb6e59470aca10687d61aa3/pset-iterators.js#L204-L213) is alternative solution that grabs the first for characters in the date string and converts it to an integer.